### ISSUE TITLE
Provide android support for FluxStore Shopify Flutter template same as provided for IOS

### ISSUE THREAD

**Issue by BinitaShah** [inspireui/support#4965]
#4826 

In ref of this issue , I am reposting same. I am not able to run your project in Android it is giving ref. error & crash in application. When I check android folder I found it blank. There is not set up done like ios. Please do the needful. I need running setup for Android.

Thanks

**Comment by minhcasi** [comment]
@BinitaShah - please help to share the detailed screenshots issues so that we could help with a suggestion - thank you!

**Comment by BinitaShah** [comment]
![1](https://user-images.githubusercontent.com/49264648/92987687-7622cd80-f4e2-11ea-9d78-769d34eca09f.png)
![2](https://user-images.githubusercontent.com/49264648/92987688-78852780-f4e2-11ea-9aaa-063f406f2e4b.png)
![3](https://user-images.githubusercontent.com/49264648/92987689-791dbe00-f4e2-11ea-95fc-c4b0f3b88cf0.png)


Hello Minhcasi,

I am telling about setup for Google Adds, Firebase (Analytics, Auth, FCM, database,Core),Google signing,Google Messaging, Google Maps, fb signing, Google Utilities and any other third party which we used & required to set up in native environment .

In the code I found that all these configuration available for IOS but not fo Android . I am an Android developer & I have not much idea about IOS.

For further details you can see attached screenshots of IOS & Android folder of the code I downloaded from your site.

In short, I want fully running applications for Android where I found all things working.

Thanks.r[](url)

**Comment by minhcasi** [comment]
@BinitaShah - if you are using the Shopify version, the social login facebook/google/sms is not ready support, please help to refer to our document guide https://docs.inspireui.com/fluxstore/sign-in/ 

Most of the config is located inside the config and common folder, or you could refer more to the library itself  https://pub.dev/packages/flutter_facebook_login

**Comment by BinitaShah** [comment]
Hello Minh,

You mentioned in your site set up for Android & IOS both will be shared in code.

**Comment by minhcasi** [comment]
Duplicate of #5009
